functions.py

```diff
@@ -0,0 +1,44 @@
+import os
+import json
+import requests
+import urllib.request
+from datetime import datetime
+
+from typing import Dict
+
+
+def getFilenames(path: str) -> list:
```

Comment:
Python style guide recommends using `snake_case` as function names :) 
https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

It's best to enable these checks in your code editor